<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportViewWidget</class>
 <widget class="QWidget" name="ReportViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>614</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QGroupBox" name="reportViewRootGroupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>171</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#reportViewRootGroupBox{ border:none }</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="reportViewGroupBox">
        <property name="styleSheet">
         <string notr="true">QGroupBox#reportViewGroupBox{ border:none }</string>
        </property>
        <property name="title">
         <string notr="true"/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="treeWidget">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="autoScroll">
            <bool>true</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="autoExpandDelay">
            <number>-1</number>
           </property>
           <property name="indentation">
            <number>20</number>
           </property>
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="headerVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="headerMinimumSectionSize">
            <number>70</number>
           </attribute>
           <attribute name="headerDefaultSectionSize">
            <number>200</number>
           </attribute>
           <column>
            <property name="text">
             <string>Calls</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Allocated</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Calls count</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="plainTextEdit">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;
   &lt;head/&gt;
   &lt;body&gt;
   // Each log entry has the following syntax:&lt;br&gt;
   //&lt;br&gt;
   // + BYTES_DELTA (NEW_BYTES - OLD_BYTES) NEW_COUNT allocs BackTrace TRACEID&lt;br&gt;
   // + COUNT_DELTA (NEW_COUNT - OLD_COUNT) BackTrace TRACEID allocations&lt;br&gt;
   // ... stack trace ...&lt;br&gt;
   //&lt;br&gt;
   // where:&lt;br&gt;
   //&lt;br&gt;
   // BYTES_DELTA - increase in bytes between before and after log&lt;br&gt;
   // NEW_BYTES - bytes in after log&lt;br&gt;
   // OLD_BYTES - bytes in before log&lt;br&gt;
   // COUNT_DELTA - increase in allocations between before and after log&lt;br&gt;
   // NEW_COUNT - number of allocations in after log&lt;br&gt;
   // OLD_COUNT - number of allocations in before log&lt;br&gt;
   // TRACEID - decimal index of the stack trace in the trace database&lt;br&gt;
   // (can be used to search for allocation instances in the original&lt;br&gt;
   // UMDH logs).
   &lt;/body&gt;
&lt;/html&gt;</string>
           </property>
           <property name="lineWrapMode">
            <enum>QPlainTextEdit::NoWrap</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="controlsGroupBox">
        <property name="maximumSize">
         <size>
          <width>241</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox#controlsGroupBox{ border:none }</string>
        </property>
        <property name="title">
         <string notr="true"/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignBottom">
          <widget class="QGroupBox" name="reportViewFilterGroupBox">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>292</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Filter</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <property name="topMargin">
             <number>9</number>
            </property>
            <item row="4" column="0">
             <widget class="QCheckBox" name="filterHideSystemCheckBox">
              <property name="text">
               <string>Hide system modules</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="filterWithSymbolsCheckBox">
              <property name="text">
               <string>With symbols</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QSpinBox" name="filterLeakedTimesSpinBox">
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QCheckBox" name="filterWithPatternCheckBox">
              <property name="text">
               <string>With patterns:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="filterLeakedBytesCheckBox">
              <property name="text">
               <string>Allocated more than Kbytes:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLineEdit" name="filterWithPatternLineEdit">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Comma-separated list &amp;quot;xxx, yyy, ...&amp;quot;,&lt;br/&gt;works via logical AND.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QCheckBox" name="filterWithoutPatternCheckBox">
              <property name="text">
               <string>Without patterns:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="filterLeakedTimesCheckBox">
              <property name="text">
               <string>Allocations count more than:</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="filterWithSourcesCheckBox">
              <property name="text">
               <string>With sources</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSpinBox" name="filterLeakedBytesSpinBox">
              <property name="maximum">
               <number>999999999</number>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLineEdit" name="filterWithoutPatternLineEdit">
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Comma-separated list &amp;quot;xxx, yyy, ...&amp;quot;,&lt;br/&gt;works via logical OR.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacerAsBorder">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignTop">
          <widget class="QGroupBox" name="reportViewSortingGroupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="title">
            <string>Sorting</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="topMargin">
             <number>9</number>
            </property>
            <item row="0" column="0">
             <widget class="QComboBox" name="sortingTypeComboBox"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="reportViewTypeGroupBox">
           <property name="title">
            <string>View type</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QRadioButton" name="reportViewPlainTextRadioButton">
              <property name="text">
               <string>Report as text</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="reportViewListRadioButton">
              <property name="text">
               <string>Report as list</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="reportViewCallsCountRadioButton">
              <property name="text">
               <string>Calls count</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="reportViewCallStackTreeRadioButton">
              <property name="text">
               <string>*Call stack tree (experimental)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>88</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="infoGroupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <property name="title">
            <string>Info by filtered data</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignBottom|Qt::AlignHCenter</set>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <property name="bottomMargin">
             <number>9</number>
            </property>
            <property name="verticalSpacing">
             <number>6</number>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="allocationsEventsLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="uniqueStacksCountLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="bytesAllocatedLabel">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filterLeakedBytesCheckBox</tabstop>
  <tabstop>filterLeakedBytesSpinBox</tabstop>
  <tabstop>filterLeakedTimesCheckBox</tabstop>
  <tabstop>filterLeakedTimesSpinBox</tabstop>
  <tabstop>filterHideSystemCheckBox</tabstop>
  <tabstop>filterWithSymbolsCheckBox</tabstop>
  <tabstop>filterWithSourcesCheckBox</tabstop>
  <tabstop>filterWithPatternCheckBox</tabstop>
  <tabstop>filterWithPatternLineEdit</tabstop>
  <tabstop>filterWithoutPatternCheckBox</tabstop>
  <tabstop>filterWithoutPatternLineEdit</tabstop>
  <tabstop>sortingTypeComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
